packages/plugins/minos-router-graphql/tests/utils.py

```diff
@@ -79,6 +79,26 @@ def get_order(self, request: Request):
         return "eu38hj32-889283-j2jjb5kl"
 
 
+class FakeQueryService2:
+    """For testng purposes."""
+
+    # noinspection PyUnusedLocal
+    @enroute.graphql.query(
+        name="order",
+        argument=GraphQLField(GraphQLString),
+        output=GraphQLString,
+        authorized=True,
+        allowed_groups=[
+            "supèr_admin",
+            "admin",
+        ],
```

Comment:
I think a simple `foo="bar"` is enough to check `kwargs`.

Reply:
Done!